docs/docs/workflows/terraform-module.md

```diff
@@ -0,0 +1,119 @@
+# Use the Terraform module
+You can manage a Constellation cluster through Terraform.
+The module package is available as part of the [GitHub release](https://github.com/edgelesssys/constellation/releases/). It consists of a convenience module for each cloud service provider (`{csp}-constellation`) that combines the IAM (`infrastructure/{csp}/iam`), infrastructure (`infrastructure/{csp}`), and constellation (`constellation-cluster`) modules.
+
+## Prerequisites
+- a Linux / Mac operating system
+- a Terraform installation of version `v1.4.4` or above
+
+## Quick setup
+The convenience module allows setting up a Constellation cluster with a single module. It's easiest to consume the module through a remote source, as shown below.
+This allows to upgrade the cluster to a newer Constellation version by simply updating the module source.
+
+1. Create a directory (workspace) for your Constellation cluster.
+  ```bash
+  mkdir c11n
+  cd c11n
+  ```
+
+1. Create a `main.tf` file to call the CSP specific Constellation module.
+
+  <tabs groupId="csp">
+
+  <tabItem value="aws" label="AWS">
+
+  ```
+  module "aws-constellation" {
+    source = "https://github.com/edgelesssys/constellation/releases/download/@@CONSTELLATION_VERSION@@/terraform-module.zip//modules/aws-constellation"
```

Comment:
```suggestion
  variable "module_version" { ... }

  module "aws-constellation" {
    source = "https://github.com/edgelesssys/constellation/releases/download/${var.module_version}/terraform-module.zip//modules/aws-constellation"
```

If we don't want the user to replace the version within the URL, we could also have it as a variable, which might be more clear to the user.

Reply:
this is a good idea! did that